src/native/NativeInstabug.ts

```diff
@@ -140,12 +142,23 @@ export interface InstabugNativeModule extends NativeModule {
   addFileAttachmentWithURLToReport(url: string, filename?: string): void;
   addFileAttachmentWithDataToReport(data: string, filename?: string): void;
   willRedirectToStore(): void;
+
+  // W3C Feature Flags
+  isW3ExternalTraceIDEnabled(): Promise<boolean>;
+
+  isW3ExternalGeneratedHeaderEnabled(): Promise<boolean>;
+
+  isW3CaughtHeaderEnabled(): Promise<boolean>;
```

Comment:
I'm not sure why we should always expect null from the native side, can you please elaborate on that ?

Reply:
@kholood-ea This could happen if a crash occurs in the Android bridge for example (we use `promise.resolve(null)` when there's an error there.

```java
@ReactMethod
public void isW3ExternalTraceIDEnabled(Promise promise){

    MainThreadHandler.runOnMainThread(new Runnable() {
        @Override
        public void run() {
            try {
                promise.resolve(InternalCore.INSTANCE._isFeatureEnabled(CoreFeature.W3C_EXTERNAL_TRACE_ID));
            }
            catch (Exception e) {
                e.printStackTrace();
                promise.resolve(null);
            }

        }

    });
}
```

Reply:
in that case I believe we should return false from the bridge rather than handling null in the js side, according to TDD default value is false and i believe we should always expect boolean

Reply:
@kholood-ea That's a good idea, let's go with that 👍🏼